src/core/sys/linux/fcntl.d

```diff
@@ -17,6 +17,24 @@ enum {
     FALLOC_FL_UNSHARE_RANGE = 0x40
 }
 
+// From asm-generic/fcntl.h
+/*
```

Comment:
Not really, I'll update tomorrow with the ddoc (will fix entire file) and add the title. Thanks!